.github/workflows/build.yml

```diff
@@ -547,8 +547,27 @@ jobs:
 
           mv "PrismLauncher-Linux-x86_64.AppImage" "PrismLauncher-Linux-${{ env.VERSION }}-${{ inputs.build_type }}-x86_64.AppImage"
 
-      - name: Package (Linux, portable)
-        if: runner.os == 'Linux'
+      - name: Package (Linux, qt6, portable)
```

Comment:
We shouldn't need to have a step specific for this runner when the only difference is the version in some library paths. I basically implemented exactly this in https://github.com/PrismLauncher/PrismLauncher/pull/3304 with glob expressions [here](https://github.com/PrismLauncher/PrismLauncher/blob/ade88d7ab0c7f60e204fd05f32603281a33e0eb8/.github/workflows/build.yml#L543-L545)